demos/flowchart.html

```diff
@@ -1540,6 +1540,33 @@ <h3>flowchart</h3>
     </pre>
     <hr />
 
+    <pre class="mermaid">
+      ---
+      title: Subgraph nodeSpacing and rankSpacing example
+      config:
+        flowchart:
+          nodeSpacing: 1
+          rankSpacing: 1
+      ---
+
+      flowchart LR
+      
+      X --> Y
+      
+      subgraph X
+        direction LR
+        A
+        C
+      end
+      
+      subgraph Y
+        direction LR
+        B
+        D
+      end
```

Comment:
Awesome for adding a demo!

Can you also add this demo (or another diagram) to the [`cypress/integration/rendering/flowchart-v2.spec.js`](https://github.com/mermaid-js/mermaid/blob/d6e738ac4cf2e208326b0ca3a8702d940cb846ec/cypress/integration/rendering/flowchart-v2.spec.js) file? This is just so we get some automated screenshots of the new feature you added :)

It should look something like:

```ts
  it('Should render subgraphs with different nodeSpacing', () => {
    imgSnapshotTest(`---
title: Subgraph nodeSpacing and rankSpacing example
---
flowchart LR
  X --> Y
  subgraph X
    direction LR
    A
    C
  end
  subgraph Y
    direction LR
    B
    D
  end
`,
      { flowchart: { nodeSpacing: 1, rankSpacing: 1 } }
    );
  });
```